libminifi/src/c2/C2Agent.cpp

```diff
@@ -686,6 +686,23 @@ C2Payload C2Agent::bundleDebugInfo(std::map<std::string, std::unique_ptr<io::Inp
       throw C2DebugBundleError("Error while writing file '" + filename + "' into the debug bundle");
     }
   }
+  if (auto node_reporter = node_reporter_.lock()) {
+    constexpr const char* MANIFEST_FILE_NAME = "manifest.json";
+    auto reported_manifest = node_reporter->getAgentManifest();
+    state::response::SerializedResponseNode manifest;
+    manifest.name = std::move(reported_manifest.name);
+    manifest.children = std::move(reported_manifest.serialized_nodes);
+    manifest.array = reported_manifest.is_array;
```

Comment:
This could be refactored to:
```suggestion
    state::response::SerializedResponseNode manifest{
      .name = std::move(reported_manifest.name),
      .children = std::move(reported_manifest.serialized_nodes),
      .array = reported_manifest.is_array
    };
```

Reply:
makes sense, we don't actually even need the variable, refactored